src/main/java/uk/gov/pay/directdebit/gatewayaccounts/model/PaymentProviderServiceId.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.pay.directdebit.gatewayaccounts.model;
+
+import uk.gov.pay.commons.model.WrappedStringValue;
+
+/**
+ * The ID used by a payment provider to refer to a service that takes
+ * payments from users
+ */
+public abstract class PaymentProviderServiceId extends WrappedStringValue {
```

Comment:
The only class that extends this is GoCardlessOrganisationId at the moment. Is the idea that there will be one for Sandbox later?

Reply:
Probably but not until a bit later when we start filling out the sandbox functionality.